_pytest/doctest.py

```diff
@@ -88,7 +88,7 @@ def setup(self):
         if self.dtest is not None:
             self.fixture_request = _setup_fixtures(self)
             globs = dict(getfixture=self.fixture_request.getfixturevalue)
-            for name, value in self.fixture_request.getfuncargvalue('doctest_namespace').items():
+            for name, value in self.fixture_request.getfixturevalue('doctest_namespace').items():
```

Comment:
I added the fix for #1898 in this PR because it was trivial and I didn't want to open a separate PR just for that, hope that's OK.